src/test/resources/pages/instructorCommentsPageForCourseWithURIEncodedSessionName.html

```diff
@@ -0,0 +1,550 @@
+                                                        <div data-toggle="tooltip" data-placement="top" title="" data-original-title="Control what instructors can view">
+                                                            Instructors</div>
+                                                    </td>
+                                                    <td><input class="visibilityCheckbox answerCheckbox" value="INSTRUCTORS" checked="checked" type="checkbox">
+                                                    </td>
+                                                    <td><input class="visibilityCheckbox giverCheckbox" value="INSTRUCTORS" checked="checked" type="checkbox">
+                                                    </td>
+                                                </tr>
+                                                
+                                            </tbody>
+                                        </table>
+                                    </div>
+                                    <textarea class="form-control" rows="3" placeholder="Your comment about this response" name="responsecommenttext" id="responseCommentAddForm-1-1-1"></textarea>
+                                </div>
+                                <div class="col-sm-offset-5">
+                                    <a href="/page/instructorFeedbackResponseCommentAdd" class="btn btn-primary" id="button_save_comment_for_add-1-1-1">Save</a>
+                                    <input class="btn btn-default" value="Cancel" onclick="hideResponseCommentAddForm(1,1,1)" type="button">
+                                    <input name="courseid" value="comments.idOfUriCharsCourse" type="hidden">
+                                    <input name="fsname" value="comments.Session with name ,?:@=+$#" type="hidden">
+                                    <input name="questionid" value="{*}" type="hidden">
+                                    <input name="responseid" value="{*}" type="hidden">
+                                    <input name="user" value="comments.idOfInstructorOfCourseWithUriChars" type="hidden">
+                                    <input name="showresponsecommentsto" value="GIVER,INSTRUCTORS" type="hidden">
+                                    <input name="showresponsegiverto" value="GIVER,INSTRUCTORS" type="hidden">
+                                </div>
+                            </form>
+                        </li>
+                    </ul>
+                </td>
+            </tr>
+            
+        </tbody>
+    </table>
+</div>
+</div>
+                </div>
+                </div>
+                
+            </div>
+            <ul class="pagination">
+                <li><a href="javascript:;">«</a></li>
+                <!--<li class=""><a
+                    href="/page/instructorCommentsPage?user=comments.idOfInstructorOfCourseWithUriChars">Drafts</a></li>-->
```

Comment:
It's commented out in the .jsp. It looks like code for an unimplemented feature (Drafts?) so I left it there in case the link will be used in the future.